src/Exceptions/InvalidUserArrayDatatypeException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace ProjectManagementApi\Exceptions;
+
+class InvalidUserArrayDatatypeException extends \Exception
```

Comment:
you should 'use Exception' at the top of the file, similar to your other exception class